### ISSUE TITLE
Nova menu gravatar image unchangeable

### ISSUE THREAD

**Issue by charlielangridge** [laravel/nova-issues#310]
I've created a new Avatar field for User, but whilst the new Avatar image is show within the user resource, it's not used in the top right of Nova (by the Logout button)

**Comment by crnkovic** [comment]
Because user's name in the top right of the menu is fetching Gravatar directly, as can be seen in `resources/views/vendor/nova/partials/user.blade.php`.

**Comment by charlielangridge** [comment]
@crnkovic - yep - hence an issue since any changes will be overwritten on the next nova update

**Comment by crnkovic** [comment]
No. This is within your app directory and not vendor directory. Unless you run nova:publish with force flag, you’re good.
